opennms-doc/guide-admin/src/asciidoc/text/database-reports/database.adoc

```diff
@@ -264,17 +264,58 @@ To get the appropriate `interface descriptor` depends on the input parameter.
-In Addition _{opennms-product-name}_ must be configured to use that _Java Trust Store_.
+To establish a secure connection to the _Measurements API_, you must import the public certificate of the running _{opennms-product-name}_ to the _Java Trust Store_.
+In addition, _{opennms-product-name}_ must be configured to use that _Java Trust Store_.
 Please follow the instructions in this <<ga-operation-ssl-opennms-trust-store,chapter>> to setup the _Java Trust Store_ correctly.
 
 In addition please also set the property `org.opennms.netmgt.jasper.measurement.ssl.enable` in `$OPENNMS_HOME\etc\opennms.properties` to `true` to ensure that only secure connections are established.
 
 WARNING: If `org.opennms.netmgt.jasper.measurement.ssl.enable` is set to `false` an accidentally insecure connection can be established to the _Measurements API_ location.
-         A SSL secured connection can be established even if `org.opennms.netmgt.jasper.measurement.ssl.enable` is set to `false`.
+         An SSL secured connection can be established even if `org.opennms.netmgt.jasper.measurement.ssl.enable` is set to `false`.
 
 [[ga-database-reports-limitations]]
 === Limitations
 
  * Only a _JDBC Datasource_ to the _{opennms-product-name} Database connection_ can be passed to a report, or no datasource at all.
    One does not have to use the datasource, though.
+
+=== Creating PDF Reports from Grafana Dashboards Using OpenNMS
+
+OpenNMS provides three templates to create a PDF report from an existing Grafana dashboard. You can also schedule and email these PDF reports to anyone:
+
+* Keep staff without access to OpenNMS informed about network performance for improved capacity planning
+* Create a permanent record of strategic information and progress over a long period of time
+
+The PDF report displays each of the panels from the specified dashboard, with one, two, or four panels per page, depending on the selected template. 
+
+NOTE: This feature requires OpenNMS and an instance of Grafana with at least one dashboard and panel. OpenNMS allows you to create a report for any Grafana dashboard, not just those created using link:http://docs.opennms.org/helm/branches/master/helm/latest/welcome/index.html[OpenNMS Helm].
+
+To create a PDF from a dashboard, follow these steps:
+
+. In the OpenNMS UI, choose `Reports>Database Reports`.
+. In the `Report Templates` area, click `Grafana Dashboard Report <Xppp>`, where `<Xppp>` represents the number of panels per page you want to display. 
+. In the `Report Parameters` area, specify the appropriate information.
+.. Note that `Grafana Endpoint` is the datasource. Select a Grafana dashboard from the drop-down list. 
+.. You can also specify CSV for report type. 
+. Click `Create Report`. 
+.. You are prompted to save the report locally or open it.
+The file is saved to a folder on the OpenNMS Server [what folder?]. 
```

Comment:
"OpenNMS" same here, usage of the {opennms-product-name} for OpenNMS Horizon / OpenNMS Meridian.